README.md

```diff
@@ -1,6 +1,15 @@
 # Sync directories app
 
+[![forthebadge](https://forthebadge.com/images/badges/powered-by-black-magic.svg)](https://forthebadge.com)
+
 ## Build
 `
 py -m PyInstaller SyncDirectories.spec --noconfirm
 `
+
+## Authors:
+- [jaanonim](https://github.com/jaanonim)
+- [n2one](https://github.com/n2oneProgrammer)
+
+## Icon:
+Made by [mwawrzen](https://github.com/mwawrzen).
```

Comment:
Idk if its a good idea if we will start in mfi

Reply:
me too
but it will be unfair if we not mention it

Reply:
ok marge it, becouse in rules , I didnt find it is inlegal